### PR TITLE
[varnish] Add more setup steps, add cache-hit improvement notes, example php code to purge urls

### PR DESCRIPTION
**Pull request by aramalipoor**

### CODE REVIEW

File: additional-services/varnish.md [hunk 1]
```diff
@@ -7,28 +7,49 @@ You can quickly deploy Varnish on AbarCloud using the following steps:
...
+8. Edit your application's existing Route to go live.
+   Navigate to Applications > Routes > my-app > Actions > Edit and select Varnish service from dropdown menu.
 
-## Customization
-The AbarCloud Varnish Cache image includes sensible pre-defined configurations for many application use cases.
+## Improve Cache-hit Rate
+There are many techniques to [increase your application's cache hit-rate](https://varnish-cache.org/docs/4.1/users-guide/increasing-your-hitrate.html#users-guide-increasing-your-hitrate).
 
-For advanced usages we highly-recommend to fork [this repository](https://github.com/abarcloud/abar-varnish) and change `default.vcl` based on your application needs.
+Pages which serve cookies will NOT be cached by default Varnish's configuration. Try to not to `Set-Cookie`s on pages you would not need them.
```
Comment (alikhajeh1) [inline additional-services/varnish.md]: Pages that serve

Comment (alikhajeh1) [reply additional-services/varnish.md]: Wonder if NOT should just be not in bold

File: additional-services/varnish.md [hunk 2]
```diff
@@ -7,28 +7,49 @@ You can quickly deploy Varnish on AbarCloud using the following steps:
...
+  * [Laravel-Varnish library](https://github.com/spatie/laravel-varnish)
+  * [Wordpress vCaching](https://wordpress.org/plugins/vcaching/)
 
 ## Purging
 When your pages are updated you must purge the old cache and force Varnish to request the new version from your application service.
 
-You must implement a purging mechanism into your application workflow, for example Wordpress uses [Varnish HTTP Purge](https://wordpress.org/plugins/varnish-http-purge/) plugin to accomplish this.
-To be able to purge URLs from your application Pod, hostname or IP of the application Pod must be added in `acl purge` section of [`default.vcl`](#customization).
+You must implement a purging mechanism into your application workflow.
+To purge URLs from your application Pod, **IP of the application Pod** must be set in `REMOTE_PURGER` environment variable of Varnish's Deployment config. Navigate to Applications > Deployments > my-varnish > Environment.
```
Comment (alikhajeh1) [inline additional-services/varnish.md]: So this needs to be updated every time a new pod is started? There's no way to do it without Pod IP?

Comment (aramalipoor) [reply additional-services/varnish.md]: Yeah, that's annoying limitation. As a workaround to the standard way it is possible to use a secret token which is set on both Varnish and app's pod and is sent as a `Header` when app is sending a PURGE request :thinking:

Comment (alikhajeh1) [reply additional-services/varnish.md]: Let's go with this just now and see how early users react

**Pull request merged**